xontrib_term_integrations/wezterm.py

```diff
@@ -14,6 +14,7 @@
 import os
 import subprocess
 import sys
+from builtins import aliases
```

Comment:
this can be imported as `from xonsh.builtins import XSH` and `XSH.aliases` can be used

Reply:
used `from xonsh.built_ins` from your other file (think `_` was deliberate to avoid some confusion?)